test/e2e/connect-wallet.test.ts

```diff
@@ -55,6 +55,35 @@ describe('Connect wallet', () => {
     await settings.checkOnSettingsPage()
   })
 
+  it('I can call client.disconnect_wallet with no prior connection and get a null response', async () => {
+    // 1101-BWAL-084 can call client.disconnect_wallet with no prior connection and get a null response
+    const disconnectWalletResponse = await vegaAPI.disconnectWallet()
+    expect(disconnectWalletResponse).toBe(null)
+  })
+
+  it('can disconnect wallet via dapp and reconnect without needing approval', async () => {
+    // 1101-BWAL-083 I can call client.disconnect_wallet after successfully calling client.connect_wallet
+    // 1101-BWAL-085 A dapp can disconnect the current active connection (not it's pre-approved status i.e. the dapp can re-instate the connection without further approval)
+    await setUpWalletAndKey()
+    const navPanel = new NavPanel(driver)
+    const settings = await navPanel.goToSettings()
+    await vegaAPI.connectWallet() //change this to assert success when connectWallet is fixed
+    await connectWallet.checkOnConnectWallet()
+    await connectWallet.approveConnectionAndCheckSuccess()
+    const connectionResult = await vegaAPI.getConnectionResult()
+    expect(connectionResult).toBe(null)
+    let originalKeys = await vegaAPI.listKeys()
+    expect(originalKeys.length).toBe(1)
+    const disconnectResponse = await vegaAPI.disconnectWallet()
+    expect(disconnectResponse).toBe(null)
+    const keysAfterDisconnect = await vegaAPI.listKeys()
```

Comment:
I think this should give an error from the browser wallet. Does it not?

Reply:
Yeah it does, test code decides to log the error but return an empty list